views/list.pug

```diff
@@ -0,0 +1,10 @@
+html
+    body
+        h1 This is my grocery list: PUG version
```

Comment:
grocery list sounds a bit dull, spice it up with like "awesome things to charge on dat credit"